embulk-core/src/main/resources/embulk/parent_first_packages.properties

```diff
@@ -40,6 +40,7 @@ edu.umd.cs.findbugs.annotations
 io.airlift.slice
 io.netty.buffer
 io.netty.util
+java
```

Comment:
@muga It's actually not "necessary" in terms of just feasibility. But while debugging, I concluded we wanted this.

Without this, even before this PR is merged, `PluginClassLoader` actually receives requests to load `java.*` classes when code in the plugin uses `java.*` classes. For example, `java.lang.Object` is requested at the very beginning. (You can experiment it without merging this PR. Just try print-hooking `PluginClassLoader#loadClass` in the current `master`.)

In case of normal `ClassLoader`, which behaves the same as "parent-first", the parent's `loadClass` is called at first. Then it does not have big problems.

But, in case of "not parent-first", `PluginClassLoader#findClass` is called at first, and it could cause performance problems. The case could be more serious when this PR is merged.